### PR TITLE
docs: remove full stops from flags

### LINKED ISSUE #830: Inconsistencies in punctuation of flag description
## Current scenario

There are some inconsistencies regarding use of periods in flag descriptions. Some flags end with period and some don't. As suggested by @craicoverflow we shouldn't use periods as the flags provided by cobra (--default and --help) don't use them too.

Screenshot from 2021-07-09 17-07-07

## Proposal

Remove full stops from description of all flags.

### PR DESCRIPTION
**Pull request by ankithans**
Closes #830 

### Description
- update charmil from v0.2.0 -> v0.6.1
- remove full stops from flags as per #830 and Punctuation rule in charmil

### Verification Steps

1. Do `go test ./pkg/cmd/root`

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation change
- [ ] Other (please specify)

### Checklist

- [ ] Documentation added for the feature
- [x] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

**Comment by wtrocki** [comment]
Nice work! Pinged docs team for review.

File: pkg/localize/locales/en/cmd/kafka_delete.en.toml [hunk 1]
```diff
@@ -27,11 +27,11 @@ $ rhoas kafka delete --id=1iSY6RQ3JKI8Q0OTmjQFd3ocFRg
 
 [kafka.delete.flag.id]
 description = 'Description for the --id flag'
-one = 'Unique ID of the Kafka instance you want to delete. If not set, the current Kafka instance will be used.'
+one = 'If unique ID not set for the Kafka instance you want to delete, the current Kafka instance will be utilised'
```
Comment (rkpattnaik780) [inline pkg/localize/locales/en/cmd/kafka_delete.en.toml:30]: We should probably rephrase it as `If a unique ID is not set for ...`. Wdyt?

Comment (rkpattnaik780) [reply pkg/localize/locales/en/cmd/kafka_delete.en.toml:30]: The existing sentence consists of a description followed by an instruction/disclaimer, the proposed description sounds bit more of an instruction and less of a description.  Wdyt?

File: pkg/localize/locales/en/cmd/serviceaccount_create.en.toml [hunk 2]
```diff
@@ -38,11 +38,11 @@ $ rhoas service-account create --file-location=./service-acct-credentials.json
 
 [serviceAccount.create.flag.name.description]
 description = 'Description for --name flag'
-one = 'Name of the service account.'
+one = 'Name of the service account'
 
 [serviceAccount.create.flag.description.description]
 description = 'Description for --description flag'
-one = "Description for the service account. Only alphanumeric characters and '-', '.', ',' are accepted."
+one = "Only alphanumeric characters and '-', '.', ',' are accepted, as service account descriptions"
```
Comment (rkpattnaik780) [inline pkg/localize/locales/en/cmd/serviceaccount_create.en.toml:45]: We should probably rephrase it too.

**Review by rkpattnaik780 — COMMENTED** [review]
NIce work on joining multi-lined descriptions :) 
Posted a few questions.

File: pkg/localize/locales/en/cmd/kafka_delete.en.toml (hunk 1 above)
Comment (ankithans) [reply pkg/localize/locales/en/cmd/kafka_delete.en.toml]: yep, done some more changes, please have a look.

File: pkg/localize/locales/en/cmd/cluster_common.en.toml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 [cluster.common.flag.kubeconfig.description]
-one = 'Location of the kubeconfig file.'
+one = 'Location of the kubeconfig file'
 
 [cluster.common.flag.offline.token.description]
 one = '''
-Provide an offline token to be used by the operator. To obtain a token, visit {{.OfflineTokenURL}}.
+Provide an offline token to be used by the operator, for getting a token, visit {{.OfflineTokenURL}}
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/cluster_common.en.toml:6]:
```suggestion
Provide an offline token to be used by the operator; to get a token, visit {{.OfflineTokenURL}}
```

File: pkg/localize/locales/en/cmd/kafka_common.en.toml [hunk 4]
```diff
@@ -8,10 +8,10 @@ one = 'no Kafka instance is currently set, use the "--id" flag or set the curren
 
 [kafka.common.flag.output.description]
 description = "Description for --output flag"
-one = 'Format in which to display the Kafka instance. Choose from: "json", "yml", "yaml"'
+one = 'Choose a format from: "json", "yml", "yaml", to display the Kafka instance'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka_common.en.toml:11]:
```suggestion
one = 'Choose a format in which to display the Kafka instance (choose from: "json", "yml", "yaml")'
```

File: pkg/localize/locales/en/cmd/kafka_common.en.toml [hunk 5]
```diff
@@ -8,10 +8,10 @@ one = 'no Kafka instance is currently set, use the "--id" flag or set the curren
...
 
 [kafkas.common.flag.output.description]
-one = 'Format in which to display the Kafka instances. Choose from: "json", "yml", "yaml"'
+one = 'Choose a format from: "json", "yml", "yaml", to display the Kafka instances'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka_common.en.toml:14]:
```suggestion
one = 'Choose a format in which to display the Kafka instances (choose from: "json", "yml", "yaml")'
```

File: pkg/localize/locales/en/cmd/kafka_consumergroup_common.en.toml [hunk 6]
```diff
@@ -1,10 +1,10 @@
 [kafka.consumerGroup.common.flag.output.description]
 description = "Description for --output flag"
-one = 'Format in which to display the consumer group. Choose from: "json", "yml", "yaml"'
+one = 'Choose a format from: "json", "yml", "yaml", to display the consumer group'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka_consumergroup_common.en.toml:3]:
```suggestion
one = 'Choose a format in which to display the consumer group (choose from: "json", "yml", "yaml")'
```

File: pkg/localize/locales/en/cmd/kafka_consumergroup_common.en.toml [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
 
 [kafka.consumerGroup.list.flag.output.description]
 description = "Description for --output flag"
-one = 'Format in which to display the consumer groups. Choose from: "json", "yml", "yaml"'
+one = 'Choose a format from: "json", "yml", "yaml", to display the consumer group'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka_consumergroup_common.en.toml:7]:
```suggestion
one = 'Choose a format in which to display the consumer group (choose from: "json", "yml", "yaml")'
```

File: pkg/localize/locales/en/cmd/kafka_delete.en.toml [hunk 8]
```diff
@@ -27,11 +27,11 @@ $ rhoas kafka delete --id=1iSY6RQ3JKI8Q0OTmjQFd3ocFRg
 
 [kafka.delete.flag.id]
 description = 'Description for the --id flag'
-one = 'Unique ID of the Kafka instance you want to delete. If not set, the current Kafka instance will be used.'
+one = 'If a unique ID is not set for the Kafka instance you want to delete, the current Kafka instance will be utilised'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka_delete.en.toml:30]:
```suggestion
one = 'Unique ID of the Kafka instance you want to delete (if not provided, the current Kafka instance will be deleted)'
```

File: pkg/localize/locales/en/cmd/kafka_describe.en.toml [hunk 9]
```diff
@@ -33,4 +33,4 @@ $ rhoas kafka describe -o yaml
 
 [kafka.describe.flag.id]
 description = 'Description for the --id flag'
-one = 'Unique ID of the Kafka instance you want to view. If not set, the current Kafka instance will be used.'
+one = 'If a unique ID is not set for the Kafka instance you want to view, the current Kafka instance will be utilised'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka_describe.en.toml:36]:
```suggestion
one = 'Unique ID of the Kafka instance you want to view (if not provided, the current Kafka instance will be displayed)'
```

File: pkg/localize/locales/en/cmd/kafka_list.en.toml [hunk 10]
```diff
@@ -30,11 +30,11 @@ $ rhoas kafka list -o json
 
 [kafka.list.flag.id]
 description = 'Description for the --id flag'
-one = 'Unique ID of the Kafka instance you want to list. If not set, the current Kafka instance will be used.'
+one = 'If a unique ID is not set for the Kafka instance you want to list, the current Kafka instance will be utilised'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka_list.en.toml:33]:
```suggestion
one = 'Unique ID of the Kafka instance you want to list (if not provided, the current Kafka instance will be listed)'
```

File: pkg/localize/locales/en/cmd/kafka_topic_common.en.toml [hunk 11]
```diff
@@ -1,8 +1,8 @@
 [kafka.topic.common.flag.output.description]
-one = 'Format in which to display the Kafka topic. Choose from: "json", "yml", "yaml"'
+one = 'Choose a format from: "json", "yml", "yaml", to display the Kafka topic'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka_topic_common.en.toml:2]:
```suggestion
one = 'Display the Kafka topic (choose from: "json", "yml", "yaml")'
```

File: pkg/localize/locales/en/cmd/kafka_topic_common.en.toml [hunk 12]
```diff
@@ -1,8 +1,8 @@
...
 
 [kafka.topic.list.flag.output.description]
-one = 'Format in which to display the Kafka topics. Choose from: "json", "yml", "yaml"'
+one = 'Choose a format from: "json", "yml", "yaml", to display the Kafka topic'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka_topic_common.en.toml:5]:
```suggestion
one = 'Display the Kafka topic (choose from: "json", "yml", "yaml")'
```

File: pkg/localize/locales/en/cmd/login.en.toml [hunk 13]
```diff
@@ -33,34 +33,34 @@ $ rhoas login --token <your-token>
...
+one = "The URL of the SSO Authentication server"
 
 [login.flag.masAuthUrl]
 description = 'Description for the --auth-url flag'
-one = "The URL of the identity.api.openshift.com Authentication server."
+one = "The URL of the identity.api.openshift.com Authentication server"
 
 [login.flag.token]
-one = "Allows you to log in using an offline token, which can be obtained at {{.OfflineTokenURL}}."
+one = "Allows you to log in using an offline token, which can be obtained at {{.OfflineTokenURL}}"
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/login.en.toml:55]:
```suggestion
one = "Log in using an offline token, which can be obtained at {{.OfflineTokenURL}}"
```

File: pkg/localize/locales/en/cmd/login.en.toml [hunk 14]
```diff
@@ -33,34 +33,34 @@ $ rhoas login --token <your-token>
...
 
 [login.flag.printSsoUrl]
 description = 'Description for the --print-sso-url'
-one = "Prints the console login URL, which you can use to log in to RHOAS from a different web browser. This is useful if you need to log in with different credentials than the credentials you used in your default web browser."
+one = "Prints the console login URL, which you can use to log in to RHOAS from a different web browser which is useful if you need to log in with different credentials than the credentials you used in your default web browser"
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/login.en.toml:59]:
```suggestion
one = "Prints the console login URL, which you can use to log in to RHOAS from a different web browser (this is useful if you need to log in with different credentials than the credentials you used in your default web browser)"
```

File: pkg/localize/locales/en/cmd/login.en.toml [hunk 15]
```diff
@@ -33,34 +33,34 @@ $ rhoas login --token <your-token>
...
 
 [login.flag.scope]
 description = 'Description for the --scope flag'
-one = 'Override the default OpenID scope. To specify multiple scopes, use a separate --scope for each scope.'
+one = 'Override the default OpenID scope, if you want to specify multiple scopes, use a separate --scope for each scope'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/login.en.toml:63]:
```suggestion
one = 'Override the default OpenID scope (to specify multiple scopes, use a separate --scope for each scope)'
```

File: pkg/localize/locales/en/cmd/registry_crud.en.toml [hunk 16]
```diff
@@ -113,11 +113,11 @@ one = 'no Service Registry instance is currently set, use the "--id" flag or set
  
 [registry.common.flag.id]
 description = 'Description for the --id flag'
-one = 'Unique ID of the Service Registry instance you want to delete. If not set, the current Service Registry instance is used.'
+one = 'If a unique ID is not set for the Service Registry instance you want to delete, the current Service Registry instance will be utilised'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/registry_crud.en.toml:116]:
```suggestion
one = 'Unique ID of the Service Registry instance you want to delete (if not provided, the current Service Registry instance will be deleted)'
```

File: pkg/localize/locales/en/cmd/registry_crud.en.toml [hunk 17]
```diff
@@ -137,19 +137,19 @@ one = 'Service Registry instance "{{.Name}}" was deleted.'
 
 [registry.cmd.flag.output.description]
 description = "Description for --output flag"
-one = 'Format in which to display the Service Registry instance. Choose from: "json", "yml", "yaml".'
+one = 'Choose a format from: "json", "yml", "yaml", to display the Registry instance'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/registry_crud.en.toml:140]:
```suggestion
one = 'Display the Registry instance (choose a from: "json", "yml", "yaml")'
```

File: pkg/localize/locales/en/cmd/serviceaccount.en.toml [hunk 18]
```diff
@@ -9,10 +9,10 @@ one = 'Use these commands to create, list, describe, delete and update service a
 
 [serviceAccount.common.flag.output.description]
 description = "Description for --output flag"
-one = 'Format in which to display the service account. Choose from: "json", "yml", "yaml"'
+one = 'Choose a format from: "json", "yml", "yaml", to display the service account'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/serviceaccount.en.toml:12]:
```suggestion
one = 'Display the service account (choose from: "json", "yml", "yaml")'
```

File: pkg/localize/locales/en/cmd/serviceaccount.en.toml [hunk 19]
```diff
@@ -9,10 +9,10 @@ one = 'Use these commands to create, list, describe, delete and update service a
...
 
 [serviceAccount.list.flag.output.description]
-one = 'Format in which to display the service accounts. Choose from: "json", "yml", "yaml"'
+one = 'Choose a format from: "json", "yml", "yaml", to display the service accounts'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/serviceaccount.en.toml:15]:
```suggestion
one = 'Format in which to display the service accounts (choose from: "json", "yml", "yaml")'
```

File: pkg/localize/locales/en/cmd/serviceaccount.en.toml [hunk 20]
```diff
@@ -59,15 +59,15 @@ one = 'you must specify a file to save the service account credentials'
 
 [serviceAccount.common.flag.fileFormat.description]
 description = 'Description for the --file-format flag'
-one = 'Format in which to save the service account credentials. Choose from: "env", "json", "properties"'
+one = 'Choose a format from: "env", "json", "properties" to save the service account credentials'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/serviceaccount.en.toml:62]:
```suggestion
one = 'Format in which to save the service account credentials (choose from: "env", "json", "properties")'
```

File: pkg/localize/locales/en/cmd/serviceaccount_create.en.toml [hunk 21]
```diff
@@ -38,11 +38,11 @@ $ rhoas service-account create --file-location=./service-acct-credentials.json
 
 [serviceAccount.create.flag.name.description]
 description = 'Description for --name flag'
-one = 'Name of the service account.'
+one = 'Name of the service account'
 
 [serviceAccount.create.flag.description.description]
 description = 'Description for --description flag'
-one = "Description for the service account. Only alphanumeric characters and '-', '.', ',' are accepted."
+one = "Only alphanumeric characters and '-', '.', ',' are accepted as service account descriptions"
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/serviceaccount_create.en.toml:45]:
```suggestion
one = "Description for the service account (only alphanumeric characters and '-', '.', ',' are valid)"
```

File: pkg/localize/locales/en/cmd/status.en.toml [hunk 22]
```diff
@@ -28,7 +28,7 @@ $ rhoas status -o json
 one = 'unknown service "{{.ServiceName}}"'
 
 [status.flag.output.description]
-one = 'Format in which to display the status of your services. Choose from: "json", "yml", "yaml"'
+one = 'Choose a format from: "json", "yml", "yaml" to display the status of your services'
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/status.en.toml:31]:
```suggestion
one = 'Format in which to display the status of your services (choose from: "json", "yml", "yaml")'
```

**Review by bhardesty — COMMENTED** [review]
Left a few wording suggestions.

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
@ankithans We need to rebase this.

**Comment by wtrocki** [comment]
Amazing work @ankithans

**Comment by wtrocki** [comment]
@bhardesty if you like to look at this again.
I will merge it tommorow

**Review by bhardesty — APPROVED** [review]
